### ISSUE TITLE
RTCMultiConnection object not clearing after participant leaves

### ISSUE THREAD

**Issue by theronakpatel** [muaz-khan/RTCMultiConnection#707]

**Comment by muaz-khan** [comment]
Just tested this on Chrome v71:
* https://rtcmulticonnection.herokuapp.com/demos/Video-Conferencing.html

Opened two tabs to setup WebRTC connection. Close one tab which removed his video and updated `connection.getAllParticipants` array as well.

Chrome removed peer-detector after 5 to 10 seconds on this page: `chrome://webrtc-internals`

i.e. Chrome's garbage collector took 5 to 10 seconds to release RTCPeerConnection encoders/decoders/io/etc.

**Comment by theronakpatel** [comment]
Yes, it updates the status but it fails in the following scenario,
keep refreshing the remote page, it will keep updating the objects! (it removes the object and adds it twice).
